src/BucketTesting/Validation/CampaignUtilizationValidator.php

```diff
@@ -105,6 +105,12 @@ private function findInconsistentFeatureToggleOccurrences(): void {
 	}
 
 	private function getCampaignNameFromFeatureToggle( string $featureToggle ): string {
-		return substr( $featureToggle, 0, strrpos( $featureToggle, '.' ) );
+		$strpos = strrpos( $featureToggle, '.' );
```

Comment:
Naming the variable `strpos` when the function is `strrpos` confused me for a moment (there is also a `strpos` function). Let's give this variable a more semantic name like `$indexOfRightmostDot` or `$lastSeparatorPosition`